baseframe/filters.py

```diff
@@ -211,3 +212,39 @@ def longdate(value):
     else:
         dt = value
     return dt.strftime('%e %B %Y')
+
+
+@baseframe.app_template_filter('date')
+def date_filter(value, format='medium'):
+    if isinstance(value, datetime):
+        if value.tzinfo is None:
+            dt = UTC.localize(value).astimezone(get_timezone())
+        else:
+            dt = value.astimezone(get_timezone())
```

Comment:
This won't work for Vidya's use case where she wants to display a project's datetimes in the project's timezone, not the user's.

Try this: If the datetime has a timezone _other_ than UTC, don't change the timezone. However, this can be hard to check for as `pytz.UTC` is _not_ a singleton.

You can check for `tzoffset` being non-zero, but that too is iffy as British time is zero offset for part of the year. This will mean that a project in that timezone will appear in the user's timezone, while all other projects will appear in the local timezone.

The other option is to add an additional `usertz=True` parameter, so the caller can ask for having their tz preserved.

Finally, this will need unit tests to confirm the code behaves as expected in these various situations.